state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -345,6 +346,40 @@ pub mod pallet {
 	#[pallet::getter(fn active_bidder)]
 	pub type ActiveBidder<T: Config> = StorageValue<_, BTreeSet<T::AccountId>, ValueQuery>;
 
+	/// Maps an delegator to an operator.
+	#[pallet::storage]
+	pub type ManagedDelegations<T: Config> =
+		StorageMap<_, Blake2_128Concat, T::AccountId, T::AccountId, OptionQuery>;
+
+	/// Maps an validator to an operator.
+	#[pallet::storage]
+	pub type ManagedValidators<T: Config> =
```

Comment:
The name sounds like it maps operators to their managed validators. 

```suggestion
	pub type ValidatorToOperator<T: Config> =
```

Similar comment re. `ManagedDelegators` above, could be `DelegatorToOperator` or something.